README.md

```diff
@@ -1,49 +1,57 @@
+Ninjachat helps boost customer retention with no glitches thereby providing a good user experience. 
 
 Generate leads, resolve customer complaints, address issues, etc with easy-to-use Ninjachat application. 
 
-As the customer traction of your website grows, you can add more agents to the application, where each agent is equipped with handling 25 simultaneous live chats.You can also restrict the number of live chats an agent can do, there by giving time for new agents to get trained before scaling up the operations. 
-
-Customise user experience tracking live chats and by building custom short cuts as well.
+As the customer traction of your website grows, you can add more agents to the application, where each agent is equipped with handling 25 simultaneous live chats.You can also restrict the number of live chats an agent can do, there by giving time for new agents to get trained before scaling up the operations. Customise user experience tracking live chats and by building custom short cuts as well.
 
 - [Get Started](https://infinity.500apps.com/ninjachat) with easy to use and configurable live chat application which can be configured not only to websites but also to social media handles like LinkedIn , Facebook , etc .[Ninjachat](https://ninjachat.com)
 
+1. Sign in to your Google account.
+
+2. Go to the Google Tag Manager home and click on the Create Account button on the right
+
+3. Enter the account as (Account name) and container name( Website URL).
+
+4. The container holds all your tags, such as the Google Analytics tag, Google Ads tag, or Ninjachat tag. Choose the target platform to web and click on the Continue button.
+
+5. Read through the GTM Terms of Service. Click Yes to accept them and continue.
+
+6. You will get a container with a code snippet. Copy the code and paste it into every page of your site in the recommended places. Click Ok to close the code snippet box and finish setting up the GTM manager and your first container.
+
+### Installation 
+Adding the Ninjachat tag to your website once you're logged in to your account with the container, it's time to add a new tag with the Ninjachat tracking code inside it. To do so, follow these steps
+
+
+1. In the Google Tag Manager Home, click on the New Tag tile.
+
+2. Click on the Tag Configuration tile and search for our community template 'Ninjachatby 500apps (Chatbot Software)' in the 'Community Template Gallery'.
+
+3. Click on ‘Add to workspace’ to select the template and add it to your workspace.
 
-- Sign in to your Google account.
+4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
 
-- Go to the Google Tag Manager home and click on the Create Account button on the right
+5. Find the randomly generated Ninjachat key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
```

Comment:
Just say api key not random generated ninjachat key

Reply:
Modified Sir